<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game 2048</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/icons/ico.png</normaloff>:/images/icons/ico.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="lblScore">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>34</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Score:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="scoreValue">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QTableWidget" name="gameGrid">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>x1</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </row>
      <row>
       <property name="text">
        <string>x2</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </row>
      <row>
       <property name="text">
        <string>x3</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </row>
      <row>
       <property name="text">
        <string>x4</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </row>
      <column>
       <property name="text">
        <string>y1</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>y2</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>y3</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>y4</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="2" column="3">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="3" column="3">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblMove">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Move:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="moveValue">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew_game"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOther">
    <property name="title">
     <string>Other</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuOther"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_game">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/rocket-launch.png</normaloff>:/images/icons/rocket-launch.png</iconset>
   </property>
   <property name="text">
    <string>New game</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/logout.png</normaloff>:/images/icons/logout.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/info.png</normaloff>:/images/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
